backend/src/main/java/ca/bc/gov/hlth/mohums/security/SecurityConfig.java

```diff
@@ -71,9 +77,11 @@ protected void configure(HttpSecurity http) throws Exception {
                 .mvcMatchers(HttpMethod.GET,"/users/**").hasRole(viewUsersRole)
                 .mvcMatchers(HttpMethod.POST, "/users/{userId}/role-mappings/**").hasRole(manageUserRolesRole)
                 .mvcMatchers(HttpMethod.POST,"/users/**").hasRole(createUserRole)
-                .mvcMatchers(HttpMethod.PUT,"/users/{userId}/groups/{groupId}").access(String.format("hasRole('%s') and hasRole('%s')", manageUserDetailsRole, manageUserGroupsRole))
                 .mvcMatchers(HttpMethod.PUT,"/users/**").hasRole(manageUserDetailsRole)
-                .mvcMatchers(HttpMethod.DELETE,"/users/{userId}/groups/{groupId}").access(String.format("hasRole('%s') and hasRole('%s')", manageUserDetailsRole, manageUserGroupsRole))
+//                .mvcMatchers(HttpMethod.PUT,"/users/{userId}/groups/{groupId}").access(String.format("hasRole('%s') and hasRole('%s')", manageUserDetailsRole, manageUserGroupsRole))
```

Comment:
Here I'm not sure if the more generic rule for `"/users/**"` will enforce having `manageUserDetailsRole` in the more specific `"/users/{userId}/groups/{groupId}"`. Do we need both of those roles for group management? I.e managing user roles reguires only one corresponding role.